### PR TITLE
feat(faq-plus): deployment script

### PR DESCRIPTION
**Pull request by formulahendry**

### CODE REVIEW

File: faq-plus/deploy.cmd [hunk 1]
```diff
@@ -0,0 +1,114 @@
+@if "%SCM_TRACE_LEVEL%" NEQ "4" @echo off
```
Comment (formulahendry) [inline faq-plus/deploy.cmd:1]: Copied from https://github.com/OfficeDev/microsoft-teams-apps-faqplus/blob/master/deploy.configuration.cmd. Just update some file path. So no need to review this file for details.

Comment (blackchoey) [inline faq-plus/deploy.cmd:1]: move this script to `configapp` or `deployment` folder?

File: faq-plus/deployment/azuredeploy.json [hunk 2]
```diff
@@ -93,14 +93,14 @@
       "metadata": {
         "description": "The URL to the GitHub repository to deploy."
       },
-      "defaultValue": "https://github.com/OfficeDev/microsoft-teams-apps-faqplus.git"
+      "defaultValue": "https://github.com/OfficeDev/TeamsFx-Samples"
```
Comment (dilin-MS) [inline faq-plus/deployment/azuredeploy.json:96]: Does `.git` need to be appended?

Comment (formulahendry) [reply faq-plus/deployment/azuredeploy.json:96]: Both are OK.

File: faq-plus/deploy.cmd (hunk 1 above)
Comment (formulahendry) [reply faq-plus/deploy.cmd:1]: Just want to keep the original code structure. We may have multiple `***app`. Then put this file in `faq-plus` folder would be better.

Comment (blackchoey) [reply faq-plus/deploy.cmd:1]: The idea is to hide as much non-teamsfx parts from sample root as possible, so people don't need to understand why there's a deploy.cmd there. I have no strong opinion on this. @zyxiaoyuer do you have any suggestion about the location of this file?

**Review by blackchoey — APPROVED** [review]

**Pull request merged**